micro_dl/cli/preprocess_script.py

```diff
@@ -82,8 +82,8 @@ def pre_process(pp_config):
         num_workers = pp_config['num_workers']
 
     # Resample images first
-    if 'resample_scale' in pp_config:
-        scale_factor = pp_config['resample_scale']
+    if 'resize' in pp_config:
```

Comment:
Would you mind adding description expected resize parameters shape and order in the readme?